src/service/ratelimit.go

```diff
@@ -258,18 +270,27 @@ func (this *service) rateLimitRemainingHeader(descriptor *pb.RateLimitResponse_D
 }
 
 func (this *service) rateLimitResetHeader(
-	descriptor *pb.RateLimitResponse_DescriptorStatus) *core.HeaderValue {
```

Comment:
nit: I personally like sticking with the existing function style already used in this file.